lib/cdo/delete_accounts_helper.rb

```diff
@@ -322,6 +322,30 @@ def delete_ai_tutor_interactions(user_id)
     @log.puts "Deleted #{count} AI Tutor Interactions" if count > 0
   end
 
+  def delete_rubric_ai_evaluations(user_id)
+    as_student = RubricAiEvaluation.where(user_id: user_id)
+    as_student_count = as_student.count
+    as_student.each(&:destroy)
+    @log.puts "Deleted #{as_student_count} RubricAiEvaluation as student" if as_student_count > 0
+
+    as_teacher = RubricAiEvaluation.where(requester_id: user_id)
```

Comment:
technically, the requester is the student when it happens via the Submit button in a level, but this still makes sense to do.

Reply:
Good point! I've updated the comments to give more context about what user and requester mean. Ready for review!